templates/constraints/yml/snippets/devToolInstall.mustache

```diff
@@ -0,0 +1,28 @@
+  # Install development tool(s)
+  - uses: devTool/install
+    with:
+{{#devCert}}
```

Comment:
Why not combine the content in one place for {{#xxx}?

Reply:
Do you mean something like the following?
```
{{#devCert}}
  # Install development tool(s)
  - uses: devTool/install
    with:
      devCert:
        trust: true
{{/devCert}}
{{#funcTool}}
  # Install development tool(s)
  - uses: devTool/install
    with:
      func:
        version: ~4.0.4670
        symlinkDir: ./devTools/func
{{/funcTool}}
```
There are apps that need both devCert and funcTool, We want they can be installed in one action.